pkg/services/featuremgmt/toggles_gen.go

```diff
@@ -474,4 +474,8 @@ const (
 	// FlagAlertingModifiedExport
 	// Enables using UI for provisioned rules modification and export
 	FlagAlertingModifiedExport = "alertingModifiedExport"
+
+	// FlagEnableNativeHTTPHistogram
+	// Enables native HTTP Histograms
+	FlagEnableNativeHTTPHistogram = "enableNativeHTTPHistogram"
```

Comment:
(see https://github.com/grafana/grafana/pull/75731/files#r1341979786)

```suggestion
	// FlagEnableNativeHistograms
	// Enables native Histograms
	FlagEnableNativeHistograms = "enableNativeHistograms"
```

Reply:
I think we want to enable each histogram with their own ff. Once we are further a head we can bundle all of them under on. I rather enable the new histogram using FF and not code releases and if we enable more histogram behind the same feature flag we are enabling using code releases. 

```
if EnableNativeHTTPHistogram || EnableNativeHistograms { }
```